daemon/src/model/cfd.rs

```diff
@@ -137,14 +137,15 @@ pub enum CfdState {
     /// The taker has requested to take a CFD, but has not messaged the maker yet.
     ///
     /// This state only applies to the taker.
+    /// TODO: unused?
     TakeRequested {
         common: CfdStateCommon,
     },
-    /// The taker sent an open request to the maker to open the CFD but don't have a response yet.
+    /// The taker sent an order to the maker to open the CFD but doesn't have a response yet.
```

Comment:
it seems that we'll need to consolidate naming throughout the daemon code. This meaning of `order` makes a lot of sense, but in many places we use "order" when it should be "offer" instead (ie. what maker is offering to the taker).

Reply:
I agree, we should invest some efforts in unifying this.